examples/reactor/src/main/java/org/apache/qpid/proton/example/reactor/EchoInputStreamWrapper.java

```diff
@@ -31,11 +31,12 @@
 
 public class EchoInputStreamWrapper extends Thread {
 
+    private static final AtomicInteger idCounter = new AtomicInteger();
```

Comment:
I don't think this really needed fixed and would have just left it as is. Its only an example, likely to count to 1. Typically having a static shared between reactors would often be incorrect.

If it were to change the variable name would be of the wrong syntax.

Reply:
Since this is an example, it should show the "best practices". I do not know what the best practices are, though. Inspiring people to use static variables does not to be the right thing. What is there now is not right either...